src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/UnsupportedIndex.java

```diff
@@ -59,6 +59,11 @@ public String name() {
         return cqlIdentifierToJsonKey(indexMetadata.getName());
       }
 
+      @Override
+      public ApiIndexType indexType() {
+        return null;
+      }
+
```

Comment:
For unsupported index, the type could be `COLLECTION` or `TEXT_ANALYSED`. Not sure how to deal with both cases.